packages/web-components/src/components/va-combo-box/va-combo-box-library.js

```diff
@@ -368,6 +370,8 @@ const noop = () => {};
     return new RegExp(find, 'i');
   };
 
+  const isDataOptGroup = option => option.getAttribute('data-optgroup') !== null;
```

Comment:
This should probably be expanded to include `&& option.getAttribute('data-optgroup') !== false`.

Reply:
now that I am giving this another look, it probably should just be `option.getAttribute('data-optgroup') === 'true';`